components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/internal/IdentityCoreServiceComponent.java

```diff
@@ -359,4 +360,30 @@ public static ServiceURLBuilderFactory getServiceURLBuilderFactory() {
 
         return serviceURLBuilderFactory;
     }
+    
+    @Reference(
+            name = "identity.organization.management.component.resident.resolver",
+            service = OrganizationUserResidentResolverService.class,
+            cardinality = ReferenceCardinality.OPTIONAL,
+            policy = ReferencePolicy.DYNAMIC,
+            unbind = "unsetOrganizationUserResidentResolverService"
+    )
+    protected void setOrganizationUserResidentResolverService(
+            OrganizationUserResidentResolverService organizationUserResidentResolverService) {
+
+        if (log.isDebugEnabled()) {
```

Comment:
Debug log check not required. Better update the license also.

Reply:
Fixed 01717332471b2c20636862ea3ff943be0905c224